font/bmfont.ksy

```diff
@@ -0,0 +1,155 @@
+meta:
+  id: bmfont
+  title: BMFont binary file
+  file-extension: fnt
+  license: CC0-1.0
+  endian: le
+  bit-endian: le
+doc-ref: https://www.angelcode.com/products/bmfont/doc/file_format.html  
```

Comment:
Adding format specification without even 1-2 paragraphs of text of where it is being used, which applications use that, what era does it refer to, what information does it convey, etc, will be almost meaningless — others who might be looking into this format won't be able to find it with search engines.

Please add top-level `doc` element with at least 1-2 paragraphs of general info about this format.

Reply:
Added a bit